lib/assets/SISScraper.rb

```diff
@@ -0,0 +1,175 @@
+require 'watir-webdriver'
+
+class SISScraper
+  ALPHA = ['A','B','C','D','E','F','G','H','I','J','K','L','M','N','O','P','Q','R','S','T','U','V','W','X','Y','Z']
+
+  #TODO: This can be moved to configs later
+  DOWNLOAD_DIR = '../../../Downloads/'
```

Comment:
Let's have DOWNLOAD_DIR be a parameter for initialize so we can avoid hardcoded paths.